query/src/main/scala/hmda/query/repository/filing/FilingComponent.scala

```diff
@@ -0,0 +1,291 @@
+package hmda.query.repository.filing
+
+import hmda.query.DbConfiguration
+import hmda.query.model.filing.LoanApplicationRegisterQuery
+import hmda.query.repository.Repository
+import slick.basic.DatabaseConfig
+import slick.jdbc.JdbcProfile
+import slick.collection.heterogeneous._
+import slick.collection.heterogeneous.syntax._
+
+trait FilingComponent { this: DbConfiguration =>
```

Comment:
Quick question to help my understanding: I'm confused because this trait has `Filing` in the name, but it seems to just be dealing with individual LARs. (Though I do see that `period` is an added field, which means that this table does have most of the information about the filing)

Does this component contain all of the filing data we need? (i.e. Do we not need to have a DB projection for the Transmittal Sheet?)